packages/now-static-build/src/index.ts

```diff
@@ -56,29 +57,43 @@ function validateDistDir(
   const isDirectory = () => statSync(distDir).isDirectory();
   const isEmpty = () => readdirSync(distDir).length === 0;
 
-  const hash = isDev
-    ? '#local-development'
-    : '#configuring-the-build-output-directory';
-  const docsUrl = `https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build${hash}`;
-
-  const info = config.zeroConfig
-    ? '\nMore details: https://zeit.co/docs/v2/platform/frequently-asked-questions#missing-public-directory'
-    : `\nMake sure you configure the the correct distDir: ${docsUrl}`;
+  const link =
+    'https://zeit.co/docs/v2/platform/frequently-asked-questions#missing-public-directory';
 
   if (!exists()) {
-    throw new Error(`No output directory named "${distDirName}" found.${info}`);
+    throw new NowBuildError({
+      code: 'NOW_STATIC_BUILD_NO_OUT_DIR',
+      message: `No Output Directory named "${distDirName}" found. ${
+        !config.zeroConfig
```

Comment:
One more thing:

I would say store this second part in a variable up top.

```
const legacyMsg = !config.zeroConfig
          ? '\nMake sure you configure the the correct `distDir`.'
          : ''
```

Also make sure to add a period since this will be displayed in the UI.

Reply:
Makes sense